src/database/migrations/z1631668396-update-triggers.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+// eslint-disable-next-line no-undef
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        if (process.env.NODE_ENV === 'test') {
+            return;
+        }
+        
+        await queryInterface.sequelize.query(`
+        CREATE OR REPLACE FUNCTION update_inflow_community_states()
+    RETURNS TRIGGER AS $$
+    declare
+        state_raised numeric(29);
+        -- state_daily_raised numeric(29);
+        n_backer bigint;
+        community_id integer;
+    BEGIN
+        SELECT id INTO community_id FROM community where "contractAddress"=NEW."contractAddress";
```

Comment:
Just thought about one thing. If this is a donation to the DAO, the address does not exist as a community. Is it null then? If so, we can avoid further updates. Also, I just opened a task for us to explore removing this triggers.

Reply:
I see. 
I added an IF on the trigger to verify the community_id, if is null, just return and don't do the further updates